exercises/practice/isbn-verifier/.meta/example.sml

```diff
@@ -0,0 +1,13 @@
+val isValid: string -> bool =
+  let
+    val excludeHyphens: char list -> char list =
+      List.filter (fn c => c <> #"-")
+    fun recurse (weight: int) (total: int) (l: char list): bool =
+      case (weight, l) of
+          (0, nil) => total mod 11 = 0
+        | (_, nil) => false
+        | (1, #"X" :: nil) => total mod 11 = 1
+        | (_, c :: rest) => Char.isDigit c andalso recurse (weight - 1) (total + weight * (Char.ord c - 48)) rest
```

Comment:
Might be helpful to mention (in a comment) that `48` is the ascii code for the number `0`.

Reply:
Added local function `digitValue` for clarity.

Replaced `48` with `Char.ord #"0"`